metadata.json

```
@@ -26,7 +26,7 @@
     },
     {
       "name": "theforeman/foreman_proxy",
-      "version_requirement": ">= 19.1.0 < 20.0.0"
+      "version_requirement": ">= 19.1.0 < 21.0.0"
```

If we want to be correct, technically there won't be a 19.1.0
```suggestion
      "version_requirement": ">= 20.0.0 < 21.0.0"
```